src/Workspaces/Remote/Core/Telemetry/ApiUsageIncrementalAnalyzerProvider.cs

```diff
@@ -94,13 +94,25 @@ public async Task AnalyzeProjectAsync(Project project, bool semanticsChanged, In
                     }
                 }
 
-                var groupByAssembly = metadataSymbolUsed.GroupBy(s => s.ContainingAssembly);
-                var apiPerAssembly = groupByAssembly.Select(g => new
+                var groupByAssembly = metadataSymbolUsed.GroupBy(symbol => symbol.ContainingAssembly);
+                var apiPerAssembly = groupByAssembly.Select(assemblyGroup => new
                 {
                     // mark all string as PII (customer data)
-                    AssemblyName = new TelemetryPiiProperty(g.Key.Identity.Name),
-                    AssemblyVersion = g.Key.Identity.Version.ToString(),
-                    Symbols = g.Select(s => s.GetDocumentationCommentId()).Where(id => id != null).Select(id => new TelemetryPiiProperty(id))
+                    AssemblyName = new TelemetryPiiProperty(assemblyGroup.Key.Identity.Name),
+                    AssemblyVersion = assemblyGroup.Key.Identity.Version.ToString(),
+                    Namespaces = assemblyGroup.GroupBy(symbol => symbol.ContainingNamespace)
+                        .Select(namespaceGroup =>
+                        {
+                            var namespaceName = namespaceGroup.Key.ToString();
```

Comment:
We could hit NRE on `ToString` since `ISymbol.ContainingNamespace` might be null.

Reply:
Ah good catch. Thanks!